src/main/java/org/fundacionjala/pivotal/objects/CreateProject.java

```diff
@@ -117,14 +117,14 @@ public void setButtonCreate() {
      * @param values Map<ProjectValues, Object>
      * @return strategyMap.
      */
-    public final Map<ProjectValues, Meth> getStrategyStepMap(final Map<ProjectValues, Object> values) {
-        final Map<ProjectValues, Meth> strategyMap = new HashMap<>();
+    public final EnumMap<ProjectValues, Steps> getStrategyStepMap(final Map<ProjectValues, String> values) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Map" rather than the implementation "EnumMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)